src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div data-qa="big"
+      class="overall">
+      <input
+       data-qa="keypress"
+       type="text"
+       class="search"
+       placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <img
+       src="images/Search.svg"
+       alt="icon"
+       class="icon"
+      >
+    </div>
+    <div data-qa="small"
+      class="overall">
```

Comment:
Name classes according to their meaning.
```suggestion
      class="search-bar">
```